ert3/config/_stages_config.py

```diff
@@ -58,19 +59,21 @@ def _ensure_input_record_mime(cls, field: str, values: Dict[str, Any]) -> str:
 
 
 class Record(_MimeBase):
-    record: str
-    location: str
+    name: str = Field(None, alias="record")
+    path: pathlib.Path = Field(None, alias="location")
+    is_executable: bool = False
```

Comment:
Why the aliases? Are we trying to be backwards compatible? I don't think we should.

Reply:
This is not a backwards compatibility issue, but more of a simplification of the internal translation of user configuration to PrefectEnsemble Configuration construction. 
For Unix step input field uses the "name" and "path" keys, plus input is the amalgamation of the user's input and transportable commands configuration. But this is not true for Function step. so I see now how this solution may actually complicate things instead of simplifying it. Thus it will be reverted back going forward